### PR TITLE
Make private beta colour a lighter orange than beta

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: assets/sass/styles.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 $backlog-colour: govuk-colour("purple");
 $discovery-colour: govuk-colour("bright-purple");
 $alpha-colour: govuk-colour("pink");
-$private-beta-colour: govuk-colour("brown");
+$private-beta-colour: #f7a074;
```
Comment (stevenalanwilson) [inline assets/sass/styles.scss:6]: @psoleckimoj  I was going to suggest _govuk-colour("light-pink")_ ... its seems fit if you look at the[ GDS colour palette. ](https://design-system.service.gov.uk/styles/colour/) ?

Comment (psoleckimoj) [reply assets/sass/styles.scss:6]: @swilson-mojhub it's not really light orange though :/ The GDS does allow 25%/50% tints, it just doesn't specify them on that page with particular styles?

Comment (stevenalanwilson) [reply assets/sass/styles.scss:6]: ah I see... coolio :)

**Review by stevenalanwilson — APPROVED** [review]
LGTM

**Review by davidthomas22 — COMMENTED** [review]
LGTM

**Pull request merged**